mods/e2140/content/ed/infantry/a04/rules.yaml

```
@@ -0,0 +1,22 @@
+ed_infantry_a04:
+	Inherits: ^EdInfantry
+	Tooltip:
+		Name: A04 Android
```

Use original name *Android A04*:

```suggestion
		Name: Android A04
```